public/app/core/components/AppChrome/MegaMenu/MegaMenu.tsx

```diff
@@ -43,7 +43,7 @@ export const MegaMenu = memo(
       .map((item) => enrichWithInteractionTracking(item, state.megaMenuDocked));
 
     if (config.featureToggles.pinNavItems) {
-      const bookmarksItem = findByUrl(navItems, '/bookmarks');
+      const bookmarksItem = findByUrl(navItems, config.appSubUrl + '/bookmarks');
```

Comment:
i think finding by id might be safer instead 👍 
```suggestion
      const bookmarksItem = navItems.find((item) => item.id === 'bookmarks');
```

Reply:
Makes sense, updated.